src/sentry/utils/snuba.py

```diff
@@ -298,7 +298,10 @@ def get_project_issues(project_ids, issue_ids=None):
         issue_ids = issue_ids[:500]
         hashes = GroupHash.objects.filter(group_id__in=issue_ids)
     else:
-        hashes = GroupHash.objects.filter(project__in=project_ids)
+        hashes = GroupHash.objects.filter(
+            project__in=project_ids,
+            group_id__isnull=False,
+        )[:500]
```

Comment:
Should maybe make this and the limit from L298 a named "constant" at this point?

Reply:
Hmm, yeah its not actually technically the same limit as the other one because one limits issue_ids and the other hashes, perhaps I should even increase the hashes one to like 5000. Either way, yeah it should probably be named as a constant.

Reply:
Whoops good point